src/Apps/Purchase/Components/PurchaseHistory.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Image,
+  Link,
+  Sans,
+  Serif,
+  StackableBorderBox,
+} from "@artsy/palette"
+import { PurchaseHistory_orders } from "__generated__/PurchaseHistory_orders.graphql"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface OrderRowProps {
+  order: any
+}
+const OrderRow = (props: OrderRowProps) => {
+  const { order } = props
+  const artwork = order.lineItems.edges[0].node.artwork
+
+  return (
+    artwork && (
+      <Box p={1}>
+        <StackableBorderBox
+          flexDirection="row"
+          alignItems="center"
+          justifyContent="space-between"
+        >
+          <Flex width="350px">
+            <Flex height="auto" alignItems="center">
+              {artwork.image.resized.url && (
+                <Image src={artwork.image.resized.url} width="55px" mr={1} />
+              )}
+            </Flex>
+            <Flex
+              flexDirection="column"
+              justifyContent="center"
+              style={{ overflow: "hidden" }}
+            >
+              <Link
+                href={`/orders/${order.internalID}/status`}
+                underlineBehavior="hover"
+              >
+                <Serif size="2" weight="semibold">
+                  {artwork.artist_names}
```

Comment:
Why alias this field below? We should just be able to use `artistNames` across the board.

Reply:
Just to be able to use `OrderApp` fixtures. We can switch back if we ended up using the new relay fixture generators. In the spirit of hackathon I may try it out if I get some extra time.

Reply:
I'd vote for copy-pasting and updating those fixtures (within your test file) and making this change vs. learning the new way or leaving as-is!